app/resources/setup_datastore.html.template

```diff
@@ -0,0 +1,17 @@
+{# Copyright 2015 Google Inc.  Licensed under the Apache License, Version   #}
+{# 2.0 (the "License"); you may not use this file except in compliance with #}
+{# the License.  You may obtain a copy of the License at:                   #}
+{#     http://www.apache.org/licenses/LICENSE-2.0                           #}
+{# Unless required by applicable law or agreed to in writing, software      #}
+{# distributed under the License is distributed on an "AS IS" BASIS,        #}
+{# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. #}
+{# See the License for the specific language governing permissions and      #}
+{# limitations under the License.                                           #}
+
+{# Template fragment for the uninitialized instance warning.                #}
+
+{% load i18n %}
+
+<div>
+  <p>This instance of Person Finder <b>has not been initialized yet</b>. Please go to the <a href="{{env.global_url}}/admin">admin page</a> and click on the <b>setup datastore</b> button.
```

Comment:
How about just directly put "Setup datastore" button here? Then you probably don't need to put the button in the admin page.